### PR TITLE
Added admins field in config and admin management methods in Identity class

### PR DESCRIPTION
**Pull request by AhmedSa-mir**

### CODE REVIEW

File: jumpscale/core/identity/__init__.py [hunk 1]
```diff
@@ -89,6 +89,22 @@ def register(self, host=None):
...
+        config = j.core.config.get_config()
+        config["threebot"]["admins"].append(name)
+        j.core.config.update_config(config)
+
+    def delete_admin(self, name):
+        config = j.core.config.get_config()
+        config["threebot"]["admins"].remove(name)
+        j.core.config.update_config(config)
+
+    def admins_list(self):
```
Comment (waleedhammam) [inline jumpscale/core/identity/__init__.py:105]: naming should be list_admin()

File: jumpscale/core/identity/__init__.py [hunk 2]
```diff
@@ -89,6 +89,22 @@ def register(self, host=None):
 
         return user.id
 
+    def get_threebot_config(self):
+        return self._threebot_data
+
+    def add_admin(self, name):
+        config = j.core.config.get_config()
+        config["threebot"]["admins"].append(name)
```
Comment (waleedhammam) [inline jumpscale/core/identity/__init__.py:97]: duplicate code i see
```
config = j.core.config.get_config()
config["threebot"]["admins"]
``` 
everywhere, maybe use at the top ?

File: jumpscale/core/identity/__init__.py (hunk 1 above)
Comment (AhmedSa-mir) [reply jumpscale/core/identity/__init__.py]: done thanks :D

File: jumpscale/core/identity/__init__.py (hunk 2 above)
Comment (AhmedSa-mir) [reply jumpscale/core/identity/__init__.py]: done

**Pull request merged**